src/ui/components/SecondaryToolbox/react-devtools/ReplayWall.ts

```diff
@@ -96,6 +122,11 @@ export class ReplayWall implements Wall {
     this.listener?.(message);
   }
 
+  async sendAtPauseId(event: string, payload: any, pauseId: PauseId) {
+    await this.waitForPauseId(pauseId);
+    return await this.send(event, payload);
+  }
```

Comment:
I guess it would be difficult to ensure that this method is called for sensitive events (at least "inspectElement") but we should probably add a comment above this and `send` below cautioning that it should be used for anything that has pause-specific data (aka `"inspectElement"`)